test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -195,30 +193,30 @@ public class ValueTypeTests {
 
 	/* fields */
 	static String typeWithSingleAlignmentFields[] = {
-		"tri:QTriangle2D;:value",
-		"point:QPoint2D;:value",
-		"line:QFlattenedLine2D;:value",
-		"i:QValueInt;:value",
-		"f:QValueFloat;:value",
-		"tri2:QTriangle2D;:value"
+		"tri:LTriangle2D;:NR",
```

Comment:
Not sure if it is obvious to everyone that `NR` means `Null Restricted`. It is better to make a comment about this in the first use of `NR` in this file.